parking_lot_src/ParkingLot/Floors.java

```diff
@@ -0,0 +1,128 @@
+package ParkingLot;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.Vector;
+
+public class Floors {
+    int floorNumber;
+    int maxSlotCount;
+    List<Slots> slotsList = new ArrayList<Slots>();
+    Vector<PriorityQueue<Integer>> slotStore = new Vector<PriorityQueue<Integer>>();
+
+    int countVacant[] = new int[4]; //  1--> TRUCK, 2 --> BIKE, 3 -->CAR
+
+    public Floors(int floorNumber, int maxSlotCount) {
+        this.floorNumber = floorNumber;
+        this.maxSlotCount = maxSlotCount;
+
+        for(int i=1;i<=maxSlotCount;i++) {
+            Slots localSlot;
+            int slotStoreSize = slotStore.size();
+            System.out.println("SlotStore Size " + slotStoreSize);
+            if(i == 1)
```

Comment:
If there is more than 2 if else, move it to switch case, If possible create a different method for this